entries/event.metaKey.xml

```diff
@@ -8,6 +8,8 @@
   <longdesc>
     <p>Returns a boolean value (<code>true</code> or <code>false</code>) that indicates whether or not the <kbd>META</kbd> key was pressed at the time the event fired.
     This key might map to an alternative key name on some platforms.</p>
+    <p>On Macintosh keyboards, the <a href="http://en.wikipedia.org/wiki/Command_key">Command key (⌘)</a> maps to the META key.</p>
+    <p>On Windows keyboards, the <a href="http://en.wikipedia.org/wiki/Windows_key">Windows key</a> maps to the META key.</p>
```

Comment:
A few lines above this we wrapped _META_ in `<kbd>` tags. I guess we should do that here too.